ts/client/storageEngine/memoryEngine.ts

```diff
@@ -84,6 +84,7 @@ export class MemoryEngine<Item extends Hashable>
         depth: number,
         items: Item[]
     ): Promise<void> {
+        // TODO Jake C-T implement
```

Comment:
@ChihChengLiang Any thoughts/tips on how this should be implemented? Used in the deposit batch sync `ts/client/features/deposit.ts:122`

Reply:
Sorry, I forget this API is unimplemented. I've written an issue in #570